ifrs17/Import/4ImportScope-TechnicalMargin.ipynb

```diff
@@ -603,7 +619,14 @@
         {
             "cell_type": "markdown",
             "source": [
-                "# Contractual Service Margin"
+                "## Reinsurance Switch Logic",
```

Comment:
I think what we do is right for linked Reinsurance, but we need to check what is the unlinked case --> second step.

Reply:
I have created issue #298